artifacts/notification-as-cloud-event.yaml

```diff
@@ -0,0 +1,286 @@
+  
+      * **Event**: An "event" is a data record expressing an occurrence and its context. Events are routed from an 
+    event producer (the source) to interested event consumers.
+    
+      * **Producer**: The "producer" is a specific instance, process or device that creates the data structure 
+    describing the CloudEvent.
+  
+      * **Source**:  The "source" is the context in which the occurrence happened. In a distributed system it might 
+    consist of multiple producers. If a source is not aware of CloudEvents, an external producer creates 
+    the CloudEvent on behalf of the source.
+  
+      * **Consumer**: A "consumer" receives the event and acts upon it. It uses the context and data to execute some 
+    logic, which might lead to the occurrence of new events.
+  
+      * **Data**:  Domain-specific information about the occurrence (i.e. the payload). This might 
+    include information about the occurrence, details about the data that was changed, or more.
+  
+    # API Functionality
+    
+    Only one endpoint/operation is provided: `POST /your_webhook_notification_url`
+    This endpoint describes the event notification received on subscription listener side when the event occurred. A detailed description of the event notification is provided in the CAMARA API design guideline document.
+
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0
+externalDocs:
+  description: Product documentation at CAMARA
+  url: https://github.com/camaraproject/
+security:
+    - notificationsBearerAuth: []
+    - {}
+
+servers:
+  - url: '{apiRoot}'
+    variables:
+      apiRoot:
+        default: https://localhost:8080
+        description: Can be any notification server address sent by the client application
+tags:
+  - name: Cloud Event
+    description: |
+      Events received on subscription listener side.
+paths:
+  /your_webhook_notification_url:
+    post:
+      tags:
+        - CAMARA Cloud Event notification endpoint
+      summary: "Cloud Event notification endpoint to notify consumer that statement of fact had occurred"
+      description: |
+        INFORMATIVE ENDPOINT: The value of this endpoint is freely declared by each client app by means of resource-based
+        subscription or instance-based subscription. /your_webhook_notification_url is 
+        just a convention naming referring to an absolute URL, indeed the one indicated by API client
+        in the triggering of the procedure (resource-based or instance-based).  In this way, it represents an absolute
+        URL, i.e.: notifications won't be sent to /event-notification/vX/your_webhook_notification_url.
+      operationId: sendEvent
+      requestBody:
+        required: true
+        content:
+          application/cloudevents+json:
```

Comment:
I think this is correct but I think someone complained about lack of support by certain code generators. We could stick to just `application/json` if it is safer

Reply:
According to [CloudEvents discussion](https://github.com/cloudevents/spec/issues/1118) format of "+json" is defined in RFC6839, so it should be supported by code generators.

 I propose to follow the CloudEvents [requirement](https://github.com/cloudevents/spec/blob/main/cloudevents/formats/json-format.md#3-envelope) for JSON format:

> Such a representation **MUST** use the media type `application/cloudevents+json`